nestkernel/dynamicloader.cpp

```diff
@@ -270,7 +270,13 @@ DynamicLoaderModule::init( SLIInterpreter* i )
     LOG( M_ERROR, "DynamicLoaderModule::init", "Could not initialize libltdl. No dynamic modules will be available." );
   }
 
-  if ( lt_dladdsearchdir( NEST_INSTALL_PREFIX "/" NEST_INSTALL_LIBDIR ) )
+  // To avoid problems due to string substitution in NEST binaries during
+  // Conda installation, we need to convert the literal to string to be able to see it as
+  // a c_str; converting that to string again will limit at the first 0-byte in the cstr.
+  // We then need to convert back to c_str since lt_dladdsearchdir needs that.
+  // see #2237 and https://github.com/conda/conda-build/issues/1674#issuecomment-280378336
```

Comment:
I would shorten the comment to be the same as the one in `slistartup.cc`.

Reply:
Done.